app/cli/cmd/attestation_add.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 func newAttestationAddCmd() *cobra.Command {
-	var name, value string
+	var name, value, kind string
 	var artifactCASConn *grpc.ClientConn
 	var annotationsFlag []string
```

Comment:
we should probably add some examples of how to use this command now. See https://github.com/chainloop-dev/chainloop/blob/90d5a932ca32a11ce0262a21e4440fa84d3f147f/app/cli/cmd/attestation_push.go#L32 for some examples